### PR TITLE
Add CuteMarkEd manifest

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

File: io.github.cloose.CuteMarkEd.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "cutemarked",
+	"rename-icon": "cutemarked",
+	"rename-desktop-file": "cutemarked.desktop",
+	"finish-args":[
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=network",
```
Comment (Alexander-Wilms) [inline io.github.cloose.CuteMarkEd.json]: Network access is needed for including images hosted online

**Comment by nedrichards** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/653

**Comment by nedrichards** [comment]
ARM fails, but I don't think it's your fault:
> <•ramcq> to me, this looks like the kde runtimes need to be rebuilt w/ the new fd.o runtime. seems like missing VFP
> <•alexlarsson> ramcq: i'm working on that
When that's done we'll run another build - this may also fix a similar issue in subsurface.

**Comment by Alexander-Wilms** [comment]
Okay, thanks

**Comment by alexlarsson** [comment]
We have a new kde runtime build now, so you can try again.

**Comment by alexlarsson** [comment]
Gah, there were some issues with the kde runtime import. But *now* it should be there.

**Comment by nedrichards** [comment]
The new build is at: https://flathub.org/builds/#/builders/1/builds/664 it's ongoing!

**Comment by Alexander-Wilms** [comment]
All builds succeeded

**Comment by alexlarsson** [comment]
This succeeded, but why the heck did the build take 9 hours???

**Comment by Alexander-Wilms** [comment]
Isn't this expected for ARM builds of WebKit?

**Comment by Barthalion** [comment]
Merged, thank you.

**Pull request closed without merging**